core/templates/dev/head/pages/base.html

```diff
@@ -81,22 +81,22 @@
         ACTIVITY_STATUS_PUBLICIZED: JSON.parse(
           '{{ACTIVITY_STATUS_PUBLICIZED|js_string}}'),
         ADDITIONAL_ANGULAR_MODULES: [],
-        ALL_CATEGORIES: JSON.parse('{{ALL_CATEGORIES|js_string}}'),
-        ALL_LANGUAGE_CODES: JSON.parse('{{ALL_LANGUAGE_CODES|js_string}}'),
+        //ALL_CATEGORIES: JSON.parse('{{ALL_CATEGORIES|js_string}}'),
```

Comment:
Please just delete these, rather than commenting.

Can we get some of the other stuff here into constants.json as well?